app/models/provisioning_template.rb

```diff
@@ -213,3 +183,9 @@ def self.pxe_default_combos
     combos
   end
 end
+
+ConfigTemplate = begin
```

Comment:
Yes, that's for compatibility with plugins. I think it's better to have such layer in Foreman itself, so not every plugin has to do it. Also I think we should try to minimize frustrating plugin devs by breaking changes without warnings, it's hard to keep up especially for bigger plugins like Katello that touches a lot of code. I'm not sure if this is going to work flawlessly, but I'd like to see if Katello tests will be green.